jax/_src/lib/__init__.py

```diff
@@ -131,9 +130,22 @@ def _parse_version(v: str) -> Tuple[int, ...]:
 
 
 # TODO(rocm): check if we need the same for rocm.
-cuda_path: Optional[str]
-cuda_path = os.path.join(os.path.dirname(jaxlib.__file__), "cuda")
-if not os.path.isdir(cuda_path):
-  cuda_path = None
+
+def _cuda_path() -> Optional[str]:
+  _jaxlib_path = pathlib.Path(jaxlib.__file__).parent
+  # If the pip package nvidia-cuda-nvcc-cu11 is installed, it should have
+  # both of the things XLA looks for in the cuda path, namely bin/ptxas and
+  # nvvm/libdevice/libdevice.10.bc
+  path = _jaxlib_path.parent / "nvidia" / "cuda_nvcc"
```

Comment:
Don't know if its intentional but you have a double parent here. First on line 135 and then on line 139.

Reply:
It is indeed intentional. The first one goes from a file name to a directory name, and the second one goes to the parent directory.